scripts/set-availability.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This script adds the `available=true` field to all mentors that
+ * have not set this field already
+ * 
+ * Usage:
+ * 
+ *  $ yarn user:availability
+ * 
+ */
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+(async () => {
+  mongoose.connect(process.env.MONGO_DATABASE_URL, { useNewUrlParser: true });
+
+
+  const User = mongoose.model('User', {
+    _id: String,
+    available: Boolean,
+    email: String,
+    roles: Array,
+  });
+
+  const users = await User.find({ roles: 'Mentor', available: undefined }).exec();
```

Comment:
Looks like we are only interested in the number of mentors without availability set. What do you think of using [`count`](https://mongoosejs.com/docs/api.html#model_Model.count) here?

```js
const mentorCount = await User.count({ roles: 'Mentor', available: undefined });
```

Reply:
Makes sense!!